jscomp/syntax/src/res_core.ml

```diff
@@ -3604,7 +3604,7 @@ and parseCallExpr p funExpr =
   in
   let isPartial =
     match p.token with
-    | DotDotDot when args <> [] ->
+    | DotDotDot ->
```

Comment:
Is there any reason for this restriction beyond the fact that `f(...)` can simple be reduced to `f` (if not piped)?